src/foraging_gui/Foraging.py

```diff
@@ -5714,6 +5720,34 @@ def _perform_backup(self, BackupSave):
         except Exception as e:
             logging.error("backup save failed: {}".format(e))
 
+    def lick_spout_bias_correction(self,
+                                   bias: float,
+                                   trial_number: int):
+
+        """
+         Evaluate and move lick spout based on bias. Negative bias correlates to left; positive, right.
+        :param bias: bias value
+        :param trial_number: trial number at which bias value was calculated
+        """
+
+        specs = self.operation_control_model.lick_spout_bias_movement
+        pos = self._GetPositions()
+        if specs and specs.trial_interval >= trial_number-self.last_bias_move and pos != self.lick_spout_origin:
+
+            if abs(bias) < specs.bias_lower_threshold: # move lick spouts back to position at start of session
+                delta_step = specs.step_size_um if bias < 0 else -specs.step_size_um
+
+            else: # move lick spouts towards unbiased side
+                delta_step = specs.step_size_um if bias >= 0 else -specs.step_size_um
+
+            if self.stage_widget is not None:
+                pos["x"] += delta_step * 10e-3
```

Comment:
Why change units here? I guess `specs.step_size_um` is in micrometers, and position is in cm? I would change the variable name of `pos` to `position_cm` just to be super explicit